apps/extension/src/ui/domains/Sign/Ethereum/riskAnalysis/RiskAnalysisDrawers.tsx

```diff
@@ -0,0 +1,149 @@
+import { Transition } from "@headlessui/react"
+import { ArrowRightIcon, ShieldNotOkIcon } from "@talismn/icons"
+import { classNames } from "@talismn/util"
+import { useSetting } from "@ui/hooks/useSettings"
+import { FC, useCallback, useEffect } from "react"
+import { useTranslation } from "react-i18next"
+import { Button, Drawer, useOpenClose } from "talisman-ui"
+
+import { RiskAnalysisRecommendation } from "./RiskAnalysisRecommendation"
+import { RiskAnalysisStateChanges } from "./RiskAnalysisStateChanges"
+import { RisksAnalysisAknowledgement } from "./RisksAnalysisAknowledgement"
+import { EvmRiskAnalysis } from "./types"
+
+const RiskAnalysisDrawerContent: FC<{ riskAnalysis: EvmRiskAnalysis }> = ({ riskAnalysis }) => {
+  const { t } = useTranslation()
+
+  return (
+    <div className="bg-grey-850 flex max-h-[60rem] w-full flex-col gap-12 rounded-t-xl p-12">
+      <div className="scrollable scrollable-700  flex-grow overflow-y-auto pr-4 text-xs  leading-[2rem]">
+        <div className="text-body-secondary leading-paragraph flex w-full flex-col gap-12">
+          <div className="text-body text-md text-center font-bold">{t("Risk Assessment")}</div>
+          <RiskAnalysisRecommendation riskAnalysis={riskAnalysis} />
+          <RiskAnalysisStateChanges riskAnalysis={riskAnalysis} />
+        </div>
+      </div>
+      <RisksAnalysisAknowledgement riskAnalysis={riskAnalysis} />
+      <div>
+        <Button onClick={riskAnalysis.review.drawer.close} className="w-full">
+          {t("Close")}
+        </Button>
+      </div>
+    </div>
+  )
+}
+
+export const RiskAnalysisPromptAutoRiskScan: FC = () => {
+  const [, setAutoRiskScan] = useSetting("autoRiskScan")
+  const { t } = useTranslation()
+
+  const handleClick = useCallback(
+    (enable: boolean) => () => {
+      setAutoRiskScan(enable)
+    },
+    [setAutoRiskScan]
+  )
+
+  return (
+    <div className="animate-fade-in bg-grey-850 flex w-full flex-col gap-12 rounded-t-xl p-12">
+      <div className="scrollable scrollable-700  flex-grow overflow-y-auto pr-4 text-xs  leading-[2rem]">
+        <div className="text-body-secondary leading-paragraph flex w-full flex-col gap-8">
+          <div className="text-body text-md text-center font-bold">
+            {t("Automatic risk assessments")}
+          </div>
+          <div className="text-body-secondary text-justify text-sm">
```

Comment:
oops, sorry - I didn't mean justify, meant center
```suggestion
          <div className="text-body-secondary text-center text-sm">
```